install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc

```diff
@@ -42,13 +42,12 @@ need to adjust these steps to fit your cluster.
       keyFile: /etc/origin/master/openshift-master.key
 ----
 
-. Start {product-title} with the updated configuration:
+. Restart {product-title} with the updated configuration:
 +
 ----
-# openshift start \
-    --public-master=https://openshift.example.com:8443 \
-    --master-config=/etc/origin/master/master-config.yaml \
-    --node-config=/etc/origin/node-node1.example.com/node-config.yaml
+/usr/local/bin/master-restart api api
```

Comment:
@vwalek, all commands need to start with $ or #. If you need to run these commands as root, use #.

Reply:
@kalexand-rh - thx, I did not know that.

Reply:
Happy to help! Thanks for opening the PR.